spec/exception_page_spec.cr

```diff
@@ -28,7 +28,7 @@ class ErrorDebuggingFlow < LuckyFlow
   end
 
   def should_be_able_to_view_other_frames
-    el("@code-frame-file", "request_processor.cr").click
-    el("@code-frame-summary", text: "request_processor.cr").should be_on_page
+    el("@code-frame-file", "server.cr").click
```

Comment:
This was because the stack trace used to say it came from https://github.com/crystal-lang/crystal/blob/master/src/http/server/request_processor.cr but now it says it comes from https://github.com/crystal-lang/crystal/blob/master/src/http/server.cr